src/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Leader board</title>
+</head>
+
+<body>
+    <h1>Leaderboard</h1>
+    <div class="flex">
+        <div id="recent-socres">
+            <h2>Recent Scores <button type="button">Refresh</button></h2>
+            <ul class="list-items">
+                <li>
+                    <p>name:100</p>
+                </li>
+                <li>
+                    <p>name:20</p>
+                </li>
+                <li>
+                    <p>name:50</p>
+                </li>
+                <li>
+                    <p>name:100</p>
+                </li>
+                <li>
+                    <p>name:20</p>
+                </li>
+                <li>
+                    <p>name:50</p>
+                </li>
+            </ul>
```

Comment:
- [ ] I would recommend adding the data dynamically from the `js` file and using proper ES6 syntax. It will help you for the next milestone as well where you have to `hit the API` as well.